public/docs/ts/latest/guide/dependency-injection.jade

```diff
@@ -84,7 +82,7 @@ include ../../../../_includes/_util-fns
 :marked
   Now we create a car by passing the engine and tires to the constructor.
 
-+makeExample('dependency-injection/ts/app/car/car-creations.ts', 'car-ctor-instantiation', 'Simple Car')(format=".")
++makeExample('dependency-injection/ts/app/car/car-creations.ts', 'car-ctor-instantiation', '')(format=".")
```

Comment:
Good point. But this passage makes no sense at all if we can't tell that we're referring to two different classes, one inheriting from the other.

If we must we can say 'app/car/car-creations.ts (SimpleCar)' and 'app/car/car-creations.ts (SuperCar)' ... or whatever is technically correct.